compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -606,13 +606,13 @@ declare_lint! {
 }
 
 declare_lint! {
-    /// The `unfulfilled_lint_expectations` lint detects lint trigger expectations
-    /// that have not been fulfilled.
+    /// The `unfulfilled_lint_expectations` lint warns if a lint expectation is
+    /// unfulfilled.
```

Comment:
nit: technically "warns" instead of "detects" is sightly worse since the first one doesn't take into user defined levels (`#![allow(unfulfilled_lint_expectations)]` would not longer make the lint warn)

Reply:
I've changed this to:

> The `unfulfilled_lint_expectations` lint detects when a lint expectation is unfulfilled.